german/ruderboot.md

```diff
@@ -11,3 +11,5 @@ Werden Sie:
 [Gefasst bleiben und ihre Kleidung ausziehen und aus dem Boot werfen?](gefasst-bleiben/gefasst-bleiben.md)
 
 [Versuchen den Finger ins Loch zu stecken?](finger-ins-loch/finger-ins-loch.md)
+
+[Gewagt mit einem h�bschen Kopfsprung ins Wasser springen](Kopfsprung/kopfsprung.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.